Packs/CloudIDS/IncidentTypes/CloudIDS.json

```diff
@@ -0,0 +1,42 @@
+[
+	{
+		"id": "Cloud IDS",
+		"version": 15,
+		"modified": "2021-07-08T14:34:09.598121557Z",
+		"sortValues": null,
+		"packID": "",
+		"itemVersion": "",
+		"fromServerVersion": "",
+		"toServerVersion": "",
+		"vcShouldIgnore": false,
+		"vcShouldKeepItemLegacyProdMachine": false,
+		"commitMessage": "",
+		"shouldCommit": false,
+		"locked": false,
+		"name": "Cloud IDS",
+		"prevName": "Cloud IDS",
+		"color": "#F8E7A5",
+		"playbookId": "IP Blacklist - GCP Firewall",
```

Comment:
Please change to: "GCP Firewall - Blacklist IP"

Reply:
Change Accordingly